graphql_compiler/tests/integration_tests/integration_test_helpers.py

```diff
@@ -63,3 +64,43 @@ def compile_and_run_sql_query(schema, graphql_query, parameters, engine, metadat
             results.append(dict(result))
         trans.rollback()
     return results
+
+
+def compile_and_run_neo4j_query(schema, graphql_query, parameters, neo4j_client):
+    """Compile and run a Cypher query against the supplied graph client."""
+    compilation_result = compile_graphql_to_cypher(
+        schema, graphql_query, type_equivalence_hints=None)
+    query = compilation_result.query
+    with neo4j_client.driver.session() as session:
+        results = session.run(query, parameters)
+    return results.data()
+
+
+def compile_and_run_redisgraph_query(schema, graphql_query, parameters, redisgraph_client):
+    """Compile and run a Cypher query against the supplied graph client."""
+    converted_parameters = {
+        name: try_convert_decimal_to_string(value)
+        for name, value in six.iteritems(parameters)
+    }
+    compilation_result = graphql_to_redisgraph_cypher(schema, graphql_query, converted_parameters)
+    query = compilation_result.query
+    result_set = redisgraph_client.query(query).result_set
+
+    # See description of result sets in RedisGraph here:
+    # https://github.com/RedisGraph/RedisGraph/issues/557
+    # We have a discrepancy between the RedisGraph docs and what actually happens.
+    column_names = result_set[0]
+    records = result_set[1:]
+
+    # redisgraph gives us back bytes, but we want strings.
+    decoded_column_names = [column_name.decode('utf-8') for column_name in column_names]
+    decoded_records = []
+    for record in records:
+        # decode if bytes, leave alone otherwise. For more info see here:
+        # https://oss.redislabs.com/redisgraph/result_structure/#top-level-members
+        decoded_record = [field.decode('utf-8') if type(field) in (bytes, bytearray) else field
```

Comment:
nit: since this is a multi-line comprehension, let's stick to the multi-line format used elsewhere in the project:
```
foo = [
    bar if baz else quux
    for bar in bars
]
```